h/views/api/index.py

```diff
@@ -19,7 +19,7 @@ def index(context, request):
     # parameter names are added, we'll need to add them here, or this view will
     # break (and get caught by the `test_api_index` functional test).
     templater = AngularRouteTemplater(
-        request.route_url, params=["id", "pubid", "user", "userid"]
+        request.route_url, params=["id", "pubid", "user", "userid", "username"]
```

Comment:
Looking forward to seeing this die this year :)

Reply:
The "Angular" part is misleading. The route parsing doesn't make use of Angular at all any more in the client, although I believe the syntax of the templates may have been inherited from Angular, which in turn [according to this post](https://stackoverflow.com/a/33182724/434243) may have been inspired by Rails.